text/english/dialog/vcchead.msg

```diff
@@ -26,7 +26,7 @@
 {130}{}{[Leave]}
 {131}{}{You call yourself a colonel, but you look and talk like a hillbilly in the bar.}
 
-{160}{}{Bring me that device, before I completely lose my patience.}
+{160}{}{Bring me that device before I completely lose my patience.}
```

Comment:
I removed many commas which were ungrammatical.  I didn't change any that were purely stylistic/arguable.